draft-wang-ppm-differential-privacy.md

```diff
@@ -456,9 +473,74 @@ class DpMechanism:
 
 > TODO: Specify a Gaussian sampler
 
-## Randomized Response {#rr}
+## Binary Randomized Response {#rr}
+
+The binary randomized response mechanism specified in Appendix C.1 of
+{{MJTB+22}} is `EPSILON_0`-DP at the Client level, in the deletion-DP model
+(Definition II.4 of {{EFMR+20}}).
+
+Binary randomized response's input is a single bit `x`. The bit is flipped to
```

Comment:
A single bit or a bit vector? I still think an illustrative example would be helpful. (What type of data might be noising? Perhaps describe the histogram use case?)

Reply:
Yup, that clarifies it, thanks.